### DISCUSSION TITLE
Studio Quantum AiR [Goethe Institut]

### DISCUSSION THREAD

**Discussion by studiofomalhaut** [Quantumland-art/community#66] [Art]
<a href="https://www.goethe.de/prj/lqs/de/index.html" target="_blank"><img width="1365" alt="Screenshot 2024-07-20 at 12 59 55" src="https://github.com/user-attachments/assets/1b8f2ded-82f9-4a97-a8b6-50a16a28f201"></a>

Studio Quantum is in its second year - it is an international artists-in-residence program from the Goethe Institut where emerging quantum technologies are explored through the lens of art. In 2024, eight residencies take place in Dublin, Johannesburg, Lagos, Mexico City, Karlsruhe, Rotterdam, Bangalore and London.

On their [website](https://www.goethe.de/prj/lqs/en/index.html) you can see infos on this year's residents, as well as other cultural and educational events related to the intersections between art and quantum technologies. 

One example is the work of VR and moving-image artist Ìfẹ́olúwa Ọ̀ṣúnkọ́yà (also known as [Skodo Lee](https://www.instagram.com/skodo_lee/) ) who spent the first month of his residency in Lagos, Nigeria, working at the [@imisi3D](https://www.instagram.com/imisi3d/) virtual reality lab and collaborating with the [Goethe-Institut Nigeria](https://www.instagram.com/gi_nigeria/) as part of the ‘Dreaming New Worlds’ programme. Afterwards, [he visited Trinity College Dublin](https://www.instagram.com/p/C4_DRTIIWpk/?img_index=3) in Ireland, to continue working on his proposal, and to collaborate with quantum physicists, scientists, artists and researchers, gaining important insights which he has incorporated into his work around the intersection of quantum physics, virtual reality and performance.

<a href="https://www.instagram.com/p/C4yGZAhshA7/?img_index=1" target="_blank"><img width="552" alt="Screenshot 2024-07-20 at 13 02 52" src="https://github.com/user-attachments/assets/389ec22e-09f3-4b40-bc78-5d7a6c9884c6"></a>

There is an open call once every year, applications can be submitted around the beginning of the year. Subscribe to Studio Quantum's [newsletter](https://www.goethe.de/prj/lqs/en/new.html#newsletter_a_73289101) to stay afloat!

**Comment by omarcostahamido** [comment]
@studiofomalhaut 
thank you! A lot to unpack here ;)

And welcome Yana!

**Reply by studiofomalhaut** [reply]
@omarcostahamido Thank you! Yes, I will be making more posts about individual artists' work, which hopefully makes it easier to take in small bite-sizes.
